scripts/convertQuizToCSV.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Export a json quiz from a file to csv quiz file, pass file name as argument
+ * @example npx ts-node convertQuizToCSV.ts ~/Downloads/quiz.json
+ * This will create a csv file in the same directory as the quiz.json file
+ * with file name = <the quiz name in the json>.csv
+ */
+const { getCSVExportContentForQuiz } = require('../src/helpers/importExport');
```

Comment:
Can we move it to backend ?

Reply:
This file is not used in the app. Just a script that I used to convert. It can be useful. So, kept it in scripts folder. It won't be included with the frontend bundle.